autorest/sender.go

```diff
@@ -222,11 +254,12 @@ func WithLogging(logger *log.Logger) SendDecorator {
 }
 
 // DelayForBackoff invokes time.After for the supplied backoff duration raised to the power of
-// passed attempt (i.e., an exponential backoff delay). Backoff may be zero. The delay may be
-// canceled by closing the passed channel. If terminated early, returns false.
+// passed attempt (i.e., an exponential backoff delay). Backoff duration is in seconds and can set
+// to zero for no delay. The delay may be canceled by closing the passed channel. If terminated early,
+// returns false.
 func DelayForBackoff(backoff time.Duration, attempt int, cancel <-chan struct{}) bool {
 	select {
-	case <-time.After(time.Duration(math.Pow(float64(backoff), float64(attempt)))):
+	case <-time.After(time.Duration(math.Pow(backoff.Seconds(), float64(attempt))) * time.Second):
```

Comment:
Backoff*(2^attempt) looks good. Its exponential and won't increase steeply. We can switch to this.

backoff^attempt for 10 seconds
10, 100, 1000,..

whereas backoff * (2^attempt)
10, 20, 40, 80,..

@brendandixon Is it look fine?